cql3/statements/create_view_statement.cc

```diff
@@ -137,6 +144,14 @@ std::pair<view_ptr, cql3::cql_warnings_vec> create_view_statement::prepare_view(
         throw exceptions::invalid_request_exception(format("Cannot create a materialized view on a table in a separate keyspace ('{}' != '{}')",
                 _base_name.get_keyspace(), keyspace()));
     }
+    const sstring& cf_name = column_family();
+    boost::regex name_regex("\\w+");
+    if (!boost::regex_match(std::string(cf_name), name_regex)) {
+        throw exceptions::invalid_request_exception(format("\"{}\" is not a valid materialized view name (must be alphanumeric character only: [0-9A-Za-z]  or the underscore)", cf_name.c_str()));
```

Comment:
nitpick: a double space in this text, and also slightly incorrect grammar - you can't say the name *is* a character, rather it contains characters. Maybe something like:
(must contain only alphanumeric or underscore characters [0-9A-Za-z_]).

Reply:
done